internal/aeintegrate/aeintegrate.go

```diff
@@ -0,0 +1,351 @@
+// Copyright 2015 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// Package aeintegrate facilitates end-to-end testing against the production Google App Engine.
+//
+// This is a specialized tool that might be used in addition to unit tests. It
```

Comment:
changed to "could" - this was when i wrote this package with the intent that other people could use it. it's internal now.